debug/site.js

```diff
@@ -9,7 +9,8 @@ var map = new mapboxgl.Map({
     hash: true
 });
 
-map.addControl(new mapboxgl.Navigation());
+// map.addControl(new mapboxgl.Navigation());
+map.addControl(new mapboxgl.Geolocate());
```

Comment:
I think this is actually useful even on a simple debug page. Maybe I would even consider adding it by default in GL JS maps if it's integrated into the zoom buttons toolbar.